test/test_manifest.json

```diff
@@ -159,6 +159,14 @@
        "type": "eq",
        "annotations": true
     },
+    {  "id": "issue14317",
+      "file": "pdfs/issue14317.pdf",
+      "md5": "581c807aaf604542b2dafc1f814dc6c8",
+      "rounds": 1,
+      "lastPage": 1,
+      "type": "eq",
+      "annotations": true
+   },
```

Comment:
Is this really necessary, since I believe that `issue13915` should cover this already?